fixtures/metadata/src/tests.rs

```diff
@@ -132,12 +132,15 @@ mod test_type_ids {
     #[test]
     fn test_user_types() {
         check_type_id::<Person>(Type::Record {
+            crate_name: "uniffi_fixture_metadata".into(),
```

Comment:
I think uniffi meta calls the same concept `module_path`, which seems yet more general than this - any reason to not use that same concept - IIUC that would mean we just use a name from a sub-module, whereas `crate_name` implies only the root?